queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/QueueDEFAULTConfiguration.cs

```diff
@@ -5,8 +5,11 @@ namespace fiskaltrust.Middleware.Localization.QueueDEFAULT
 {
     public class QueueDEFAULTConfiguration
```

Comment:
Add an explanation that different queue configuration stuff can be added to this class if needed by the market